administrator/components/com_joomlaupdate/src/Model/UpdateModel.php

```diff
@@ -557,31 +575,26 @@ public function createRestorationFile($basename = null)
 		$app->setUserState('com_joomlaupdate.password', $password);
 		$app->setUserState('com_joomlaupdate.filesize', $filesize);
 
-		$data = "<?php\ndefined('_AKEEBA_RESTORATION') or die('Restricted access');\n";
-		$data .= '$restoration_setup = array(' . "\n";
+		$data = "<?php\ndefined('_JOOMLA_UPDATE') or die('Restricted access');\n";
+		$data .= '$extractionSetup = [' . "\n";
 		$data .= <<<ENDDATA
-	'kickstart.security.password' => '$password',
-	'kickstart.tuning.max_exec_time' => '5',
-	'kickstart.tuning.run_time_bias' => '75',
-	'kickstart.tuning.min_exec_time' => '0',
-	'kickstart.procengine' => 'direct',
-	'kickstart.setup.sourcefile' => '$file',
-	'kickstart.setup.destdir' => '$siteroot',
-	'kickstart.setup.restoreperms' => '0',
-	'kickstart.setup.filetype' => 'zip',
-	'kickstart.setup.dryrun' => '0',
-	'kickstart.setup.renamefiles' => array(),
-	'kickstart.setup.postrenamefiles' => false
+	'security.password' => '$password',
+	'setup.sourcefile' => '$file',
+	'setup.destdir' => '$siteroot',
 ENDDATA;
 
-		$data .= ');';
+		$data .= '];';
 
 		// Remove the old file, if it's there...
-		$configpath = JPATH_COMPONENT_ADMINISTRATOR . '/restoration.php';
+		$configpath = JPATH_COMPONENT_ADMINISTRATOR . '/update.php';
 
 		if (File::exists($configpath))
 		{
-			File::delete($configpath);
+			if (!File::delete($configpath))
+			{
+				@unlink($configpath);
+				File::invalidateFileCache($configpath);
```

Comment:
```suggestion
				File::invalidateFileCache($configpath);
				@unlink($configpath);
```
I try it with a suggestion now as my previous comments were lost somehow.

Reply:
I've already fixed that. Let me commit and push.

Reply:
Now it is here on GitHub. Before it was not.